app/src/main/java/org/medfordhistorical/overthemystic/DisplayBikeTourActivity.java

```diff
@@ -0,0 +1,13 @@
+package org.medfordhistorical.overthemystic;
+
+import android.support.v7.app.AppCompatActivity;
+import android.os.Bundle;
+
+public class DisplayBikeTourActivity extends AppCompatActivity {
```

Comment:
probably want to rename this activity to just TourSelectionActivity or something like that since this screen would be where they select what locations they want to visit.